### ISSUE TITLE
Java error

### ISSUE THREAD

**Issue by smaragdus** [libre-tube/LibreTube#1444]
### Steps to reproduce

1. Go to Settings
2. Go to Appearance
3. Change Icon
4. There is an error

### Expected behavior

No error

### Actual behavior

Error message

### LibreTube version

0.6.0

### Android version

Android 11

### Other details

java.lang.IllegalArgumentException: Component class com.github.libretube.activities.MainActivity does not exist in com.github.libretube
	at android.os.Parcel.createExceptionOrNull(Parcel.java:2421)
	at android.os.Parcel.createException(Parcel.java:2396)
	at android.os.Parcel.readException(Parcel.java:2379)
	at android.os.Parcel.readException(Parcel.java:2321)
	at android.content.pm.IPackageManager$Stub$Proxy.setComponentEnabledSetting(IPackageManager.java:6793)
	at android.app.ApplicationPackageManager.setComponentEnabledSetting(ApplicationPackageManager.java:2714)
	at com.github.libretube.ui.preferences.AppearanceSettings$$ExternalSyntheticLambda2.onPreferenceChange(R8$$SyntheticClass:11)
	at androidx.preference.Preference.callChangeListener(Unknown Source:4)
	at com.github.libretube.ui.views.MaterialPreferenceFragment$$ExternalSyntheticLambda0.onClick(R8$$SyntheticClass:6)
	at androidx.appcompat.app.AlertController$AlertParams$3.onItemClick(AlertController.java:1)
	at android.widget.AdapterView.performItemClick(AdapterView.java:330)
	at android.widget.AbsListView.performItemClick(AbsListView.java:1228)
	at android.widget.AbsListView$PerformClick.run(AbsListView.java:3220)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:368)
	at android.app.ActivityThread.main(ActivityThread.java:7710)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:954)
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.pm.PackageManagerService.setEnabledSetting(PackageManagerService.java:21252)
	at com.android.server.pm.PackageManagerService.setComponentEnabledSetting(PackageManagerService.java:21101)
	at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:3207)
	at com.android.server.pm.PackageManagerService.onTransact(PackageManagerService.java:4368)
	at android.os.Binder.execTransactInternal(Binder.java:1154)

    
[LibreTube-log.txt](https://github.com/libre-tube/LibreTube/files/9690816/LibreTube-log.txt)

![Screenshot_20221001-200431](https://user-images.githubusercontent.com/9025757/193420604-cbe53d50-5762-4dcd-abb8-a2f3a4177355.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by smaragdus** [comment]
I forgot to mention that I am using [F-Droid relrease](https://f-droid.org/en/packages/com.github.libretube/)  (**com.github.libretube_19.apk**)

**Comment by Bnyro** [comment]
Thanks for submitting, the issue is already know since #1360 and will be fixed with the next update :)

**Comment by smaragdus** [comment]
@Bnyro 

Thanks for the quick fix! I have just tested version [0.6.1](https://github.com/libre-tube/LibreTube/releases/tag/v0.6.1) , no error messages, all icons load fine (actually I like the default one best).

Is it acceptable that I ask a minor question here without opening a new issue?

**Comment by Bnyro** [comment]
> Is it acceptable that I ask a minor question here without opening a new issue?
>
I would prefer communicating normal stuff via [the Matrix room](https://matrix.to/#/#LibreTube:matrix.org), but here is fine too, sure!

**Comment by smaragdus** [comment]
@Bnyro 

I have a vague impression that in earlier versions of LibreTube there was on option which page to be loaded on app start, Home, Subscriptions, Library, etc. I browsed through the settings several times but I could not find such an option, so I am not sure whether such an option has ever existed or perhaps it appears only on initial setup after the installation?

In short- I would like to be able to set Library or Subscriptions to load after the app starts in stead of Home.

If I am not missing something and if you think that adding such an option would be useful I will open a new issue.

> I would prefer communicating normal stuff [the Matrix room](https://matrix.to/#/#LibreTube:matrix.org)  room, but here is fine too, sure!

Currently I don't have a [Matrix client](https://matrix.org/clients/) installed but when I get one I will use the room you mentioned for future questions, thanks.

**Comment by Bnyro** [comment]
> I have a vague impression that in earlier versions of LibreTube there was on option which page to be loaded on app start, Home, Subscriptions, Library, etc. I browsed through the settings several times but I could not find such an option, so I am not sure whether such an option has ever existed or perhaps it appears only on initial setup after the installation?
> 
Indeed, there was such a function, but it got removed at version 0.6.0, here the reasons why I removed:
1. There's now a navigation bar option in the appearance settings that lets you reorder the items in the navigation bar -> The first item in the order will become your default tab.
2. In the previous way of setting the default tab, the items were hardcoded and therefore adding new items would have broken it.
3. The option allowed to set tabs as default you didn't even set as visible.
> In short- I would like to be able to set Library or Subscriptions to load after the app starts in stead of Home.
If I am not missing something and if you think that adding such an option would be useful I will open a new issue.
> 
In theory it would be possible to readd such an option (of course implemented in a different way this time to prevent the previously mentioned issues). If having the first tab as default isn't sufficient for you, feel free to open a new issue then :)
